.github/workflows/build-wheels.yml

```diff
@@ -100,7 +100,7 @@ jobs:
     name: ${{ matrix.name }} (torch v${{ matrix.torch-version }})
     strategy:
       matrix:
-        torch-version: ['1.11', '1.12', '1.13', '2.0', '2.1', '2.2']
```

Comment:
I the minimal torch version listed somewhere in the docs and/or in the `pyproject.toml`?

Reply:
I don't think so, I can add it. Should I pull this commit into a separate PR?

Reply:
Yes another PR is fine. If you want create an issue that we don't forget.

Reply:
Too late, I've done it in this one =)